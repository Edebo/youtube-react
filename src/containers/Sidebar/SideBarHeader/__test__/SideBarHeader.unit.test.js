import React from "react";

import { shallow } from "enzyme";

import { SideBarHeader } from "../SideBarHeader";

describe("SideBarHeader", () => {
  test("testing empty side bar header", () => {
    const wrapper = shallow(<SideBarHeader />);

    expect(wrapper).toMatchSnapshot();
  });

  test("rendering side bar with props.title='/'", () => {
    const wrapper = shallow(<SideBarHeader title='/' />);

    expect(wrapper).toMatchSnapshot();
  });

  test("testing sidebar header with title props", () => {
    const wrapper = shallow(<SideBarHeader title='Library' />);

    expect(wrapper).toMatchSnapshot();
  });
});
