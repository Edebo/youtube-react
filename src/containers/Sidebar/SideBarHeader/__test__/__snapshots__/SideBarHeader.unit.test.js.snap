// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarHeader rendering side bar with props.title='/' 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    /
  </MenuHeader>
</MenuItem>
`;

exports[`SideBarHeader testing empty side bar header 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SideBarHeader testing sidebar header with title props 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    LIBRARY
  </MenuHeader>
</MenuItem>
`;
